### DISCUSSION TITLE
Welcome to mui-rff Discussions!

### DISCUSSION THREAD

**Discussion by lookfirst** [lookfirst/mui-rff#355] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wenscl** [comment]
Hi! Is there a way to display the radios in a row? I tried passing the property 'row' to the Radio component but it didn't work

**Reply by lookfirst** [reply]
Great question! The [other sandbox example](https://codesandbox.io/s/react-final-form-material-ui-example-tqv09) shows how to do this easily.

**Reply by lookfirst** [reply]
Specifically, this line: https://codesandbox.io/s/react-final-form-material-ui-example-tqv09?file=/src/index.jsx:1670-1710

**Reply by wenscl** [reply]
Thanks! That works perfect

**Reply by wenscl** [reply]
Me again! Now I need to set the 'clearable' prop on KeyboardDatePicker which shows the clear button. Is there a prop for that?

**Reply by lookfirst** [reply]
Hey! Welcome back. That's a really good question. Maybe a bug in the component? I reviewed the MUI code and it properly passes all the props through, you can see how it passes `okLabel` on correctly...

![image](https://user-images.githubusercontent.com/85355/102135780-5070d180-3e0d-11eb-8dbd-995833de5825.png)
